test/albumTest.js

```diff
@@ -0,0 +1,69 @@
+const chai = require('chai'),
+  chaiHttp = require('chai-http'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  expect = chai.expect,
+  should = chai.should(),
+  nock = require('nock'),
```

Comment:
you didn't added this to the devDependencies of the project, that's why the CI is failing.